modules.d/00sh/module-setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# This file is part of dracut.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Module dependency requirements.
+depends() {
+    # priority order
+    shells='dash bash busybox'
```

Comment:
I would prefer `bash` to be the most preferred option, followed by `dash` and then `busybox`.